### PR TITLE
Document installing new R pkgs and deploying changes manually

### PR DESCRIPTION
**Pull request by GeorgianaElena**
New README section about how to install additional R pkgs and how to manually deploy the changes to an existing cluster.

### CODE REVIEW

**Comment by choldgraf** [comment]
how much of the docs preceding this section are relevant to this hub, vs. boilerplate config information for HubPloy? @yuvipanda do you know?

**Comment by yuvipanda** [comment]
I think it's *all* boilerplate and can be removed!

We can also move to markdown in the same time if you'd like.

**Comment by choldgraf** [comment]
I'm a big fan of both of those options :-)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+[devtools](https://www.r-project.org/nosvn/pandoc/devtools.html)
+
+- Add the pkgs available on [Cran](https://cran.r-project.org) to the `cran_packages`
+  list. These packages will be installed using `devtools::install_version`.
+- Add the pkgs that are only available on GitHub to the `github_packages` list/
+  These packages will be installed using `devtools::install_github`.
+
+3. Pin every package to their current versions, available on the
+[R Studio package manager](https://packagemanager.rstudio.com/client/#/repos/1/packages)
+or to a specific wanted version. If the package is only available on Cran, pin to a GitHub commit hash.
```
Comment (yuvipanda) [inline README.md]:
```suggestion
or to a specific wanted version. If the package is only available on GitHub, pin to a GitHub commit hash.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+4. Build the Docker image locally and make sure everything is ok.
+
+   ```bash
+      docker build . inside the image directory
+   ```
+
+5. Commit the changes on GitHub, for `hubploy build <hub-name> --push --check-registry` to work,
```
Comment (yuvipanda) [inline README.md]:
```suggestion
5. Commit the changes to git, for `hubploy build <hub-name> --push --check-registry` to work,
```

**Comment by yuvipanda** [comment]
Thanks a lot, @GeorgianaElena!

**Pull request merged**